### DISCUSSION TITLE
Explainer ModelConfig - Different task levels

### DISCUSSION THREAD

**Discussion by acperson** [pyg-team/pytorch_geometric#7635] [Q&A]
Hi, 

I am new to pytorch geometric and have a question about the task_level parameter in ModelConfig for the explainer. 
In particular, I was wondering what the difference between node and edge task_levels are i.e. where they are called within the Explanainer Algorithm and how they influence the explanation. 
I was looking through the source code but could not find where the task_level parameter is inferred. 

Thank you so much for your help!

**Comment by rusty1s** [comment]
In most cases, the PyG explainers are agnostic to the underlying task, and this argument is just not used. For example, `GNNExplainer` does not require this argument at all. In other explainers, we use `TaskLevel.node` for example to reduce the the graph to k-hop subgraphs around nodes to explain.
